docs/use_cases/otel/representing_otel_data.md

```diff
@@ -0,0 +1,117 @@
+# Representing OTel Data
+
+OpenTelemetry data is typically structured as nested JSON, which we can model into custom types in Feldera based on the [OTel Protobuf definitions](https://github.com/open-telemetry/opentelemetry-proto/tree/main/opentelemetry/proto). While it is possible to represent the entire OTel JSON data as a `VARIANT` type ([VARIANT docs](https://docs.feldera.com/sql/json#the-variant-type)), it is often more manageable to use custom types for easier querying.
+
+```sql
+CREATE TYPE KeyValue AS (
+    key VARCHAR,
+    value VARIANT
```

Comment:
explain that a VARIANT Is still used because it can behave like a union type.

Reply:
The value is a JSON. Something like this: `{"stringValue": "some value"}`.